lib/ansible/module_utils/docker_common.py

```diff
@@ -59,21 +59,26 @@
 DEFAULT_DOCKER_HOST = 'unix://var/run/docker.sock'
 DEFAULT_TLS = False
 DEFAULT_TLS_VERIFY = False
+DEFAULT_TLS_HOSTNAME = 'localhost'
 MIN_DOCKER_VERSION = "1.7.0"
+DEFAULT_SSL_VERSION = "1.0"
+try:
+    DEFAULT_TIMEOUT = DEFAULT_TIMEOUT_SECONDS
```

Comment:
Defaults should be constant. Otherwise they cannot be properly documented, as the value to document would vary based on the dependencies installed. Since this is already documented as 60 seconds, the constant default below is all that is needed.